docs/src/main/sphinx/release/release-374.md

```diff
@@ -0,0 +1,57 @@
+# Release 374 (16 Mar 2022)
+
+## General
+
+* Improve performance of queries with `GROUP BY` clauses. ({issue}`11392`)
+* Improve performance of `GROUP BY` with a large number of groups. ({issue}`11011`)
+* Improved handling for queries where individual tasks require lots of memory.
+  If the task fails due to out-of-memory error it is retried with a full node at
+  its disposition. Mechanism is only active if task based retries are enabled
+  with `retry-policy` is set to `TASK`. ({issue}`10432`)
+* Produce better query plans by improving cost-based-optimizer estimates in the
+  presence of correlated columns. ({issue}`11324`)
+* Allocate a new Slice instead of returning a view when decoding
```

Comment:
@losipiuk @linzebing @arhimondr .. can we cut this down to be a shorter somehow?

Reply:
Maybe the following two bullets:
- Reduced unnecessary memory copying when consuming data from exchanges.
- Fixed memory accounting of pages involving `VariableWidthBlocks`s

Reply:
> Fixed memory accounting of pages involving `VariableWidthBlocks`

VariableWidthBlock is an implementation detail that users won't understand. It should be described in higher level terms, such as:

* Fix memory accounting for queries involving certain variable-width data types such as `varchar`, `varbinary`, etc.

Reply:
Updated